src/main.rs

```diff
@@ -122,7 +122,7 @@ where
     let raw_mode = opts.raw;
 
     let network_utilization = Arc::new(Mutex::new(Utilization::new()));
-    let ui = Arc::new(Mutex::new(Ui::new(terminal_backend, opts.render_opts)));
+    let ui = Arc::new(Mutex::new(Ui::new(terminal_backend, opts.render_opts, opts.interface)));
```

Comment:
If we already have a render_opts, maybe we can make the interface name part of it?